.github/workflows/ci.yaml

```diff
@@ -0,0 +1,26 @@
+---
+ name: ci
+ on:
+   - push
+ jobs:
+   lint:
+     name: lint
+     runs-on: ubuntu-latest
+     steps:
+       - uses: actions/checkout@v2
+       - name: golangci-lint
+         uses: golangci/golangci-lint-action@v2
```

Comment:
If you want, you can upgrade it to its latest release (I think v4). Another option is adding Dependabot.